internal/drivers/google/driver.go

```diff
@@ -244,6 +244,26 @@ func (p *config) Create(ctx context.Context, opts *types.InstanceCreateOpts) (in
 	return &instanceMap, nil
 }
 
+func (p *config) SetTags(ctx context.Context, instanceID string, tags map[string]string) error {
+	logr := logger.FromContext(ctx).
```

Comment:
Can we return immediately if tags are empty? We don't need to do a get if there are no tags

Reply:
Added the check in manager.go